lambda/query-user-services/query-user-services.ts

```diff
@@ -1 +1,97 @@
-export const handler = async (): Promise<void> => {};
+import { SQSEvent, SQSRecord } from "aws-lambda";
+import { Service, TxmaEvent, UserData, UserRecordEvent } from "./models";
+import { ValidationError } from "./errors";
+import {
+  SendMessageCommand,
+  SendMessageRequest,
+  SQSClient,
+} from "@aws-sdk/client-sqs";
+import { DynamoDBDocumentClient, GetCommand } from "@aws-sdk/lib-dynamodb";
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+
+const { TABLE_NAME } = process.env;
+const QUEUE_URL = "";
+const { AWS_REGION } = process.env;
+const marshallOptions = {
+  convertClassInstanceToMap: true,
+};
+const translateConfig = { marshallOptions };
+
+const dynamoClient = new DynamoDBClient({});
+const dynamoDocClient = DynamoDBDocumentClient.from(
+  dynamoClient,
+  translateConfig
+);
+
+export const parseRecordBody = (body: string): TxmaEvent => {
+  return JSON.parse(body) as TxmaEvent;
+};
```

Comment:
This is totally take-it-or-leave it, just 🤔 about where we might get duplication with typescript defs

Reply:
One of the bits of feedback I got from Paul J was that if our parsing function is just calling out to `JSON.parse` then we don't need a separate parsing function, and I've come around to agree with him.

`JSON.parse` is a well established, stable part of the Javascript core libraries. We can trust that it can take a JSON encoded string and return an object, or throw an error if that string can't be parsed. So I'd argue that we don't need to explicitly unit test the parsing at all (but it's still implicitly covered by the handler tests). If we don't need to unit test the function then I think that's an argument that we don't need a function.

The second argument is that the `as TxmaEvent` or `: TxmaEvent` return types aren't doing anything at runtime - there's no validation that the `any` type object returned by `JSON.parse` has the attributes we expect from the interface since all the types and interfaces are dropped when it's transpiled to Javascript. That's why we still need the validation steps.

So I think we can get rid of this function and change line 88 in the handler function to 
```typescript
const txmaEvent: TxmaEvent = JSON.parse(record.body);
```
and we'll simplify our code and not lose any functionality. 

(also this is the smallest of nitpicks, so if you think I'm being too pedantic then feel free to ignore me)